temporal-testing-junit4/src/main/java/io/temporal/testing/TestWorkflowRule.java

```diff
@@ -59,13 +61,19 @@
  */
 public class TestWorkflowRule implements TestRule {
 
+  public static final String TEMPORAL_SERVICE_ADDRESS = System.getenv("TEMPORAL_SERVICE_ADDRESS");
+  // Only enable when USE_DOCKER_SERVICE is true
+  public static final Boolean USE_EXTERNAL_SERVICE =
+      Boolean.parseBoolean(System.getenv("USE_DOCKER_SERVICE"));
+  public static final String NAMESPACE = "UnitTest";
```

Comment:
All above fields belong to SDKTestWorkflowRule as they are specific to SDK testing. 
TestWorkflowRule is a general-purpose class to be used by developers that use Temporal SDK.